app/models/medication.rb

```diff
@@ -43,4 +44,8 @@ def active_reminders
 
     [refill_reminder, take_medication_reminder].select(&:active?)
   end
+
+  def weekly?
+    weekly_dosage.count != 7
```

Comment:
The name of the method and the logic inside of it is contradictory to me. If we want to keep the name `weekly?`, then it should be `weekly_dosage.count == 7`. If not, `weekly?` should be renamed to `not_weekly?`.

Reply:
I think your suggestion works! Thanks for explaining!